mlflow/ml-package-versions.yml

```diff
@@ -5,15 +5,15 @@ sklearn:
       pip install git+https://github.com/scikit-learn/scikit-learn.git
 
   models:
-    minimum: "0.24.1"
+    minimum: "1.2.0"
```

Comment:
MLR 11.3 uses 0.24.2, how do we determine the minimum version here?

Reply:
I roughly checked the released time to decide it.

Reply:
Do we want new mlflow version support MLR 11.3 ?

Reply:
I'm not sure, but we definitely want to test packages that should be supported in case users use latest mlflow in old runtime cc @harupy

Reply:
This is strict, DBR 10.4 LTS and 9.1 LTS are still in life-cycle, shall we make them supported too ?

Reply:
Well these two versions are going to deprecate soon so I feel like fine.

Reply:
Updated, now the updated flavor versions cover the versions which are preinstalled in MLR runtime 9.1 LTS ~ 14.3 LTS